napari/_tests/test_function_widgets.py

```diff
@@ -1,11 +1,12 @@
 import numpy as np
-from qtpy.QtWidgets import QDockWidget
 
 import napari.layers
 
 
 def test_add_function_widget(make_test_viewer):
     """Test basic add_function_widget functionality"""
+    from qtpy.QtWidgets import QDockWidget
```

Comment:
Why did this have to move? I don't mind, but just curious?

Reply:
if qtpy is not install, with it in the head of the file you'll get an error during test collection: 
```
================================= short test summary info ==================================
ERROR napari/_tests/test_function_widgets.py
!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!
```

with it inside the function, tests will run, you'll just get a failure for that test... which is actually kinda nice cause it gives a hint of what other tests still succeed (and therefore might not depend on qt):
```
======================================== test session starts =========================================
platform darwin -- Python 3.8.6, pytest-6.2.1, py-1.10.0, pluggy-0.13.1
PyQt5 5.15.2 -- Qt runtime 5.15.2 -- Qt compiled 5.15.2
rootdir: /Users/talley/Dropbox (HMS)/Python/forks/napari, configfile: pyproject.toml
plugins: napari-plugin-engine-0.1.8, ordering-0.6, cov-2.10.1, repeat-0.9.1, timeout-1.4.2, qt-3.3.0
collected 167 items

napari/_tests/test_viewer.py F                                                                 [  0%]
napari/_tests/test_adding_removing.py EEEEEEEEEEEEEEEEE                                        [ 10%]
napari/_tests/test_advanced.py EEEEEEEE                                                        [ 15%]
napari/_tests/test_cli.py FFFF.F                                                               [ 19%]
napari/_tests/test_draw.py E                                                                   [ 19%]
napari/_tests/test_dtypes.py EEEEEEEEEEEE                                                      [ 26%]
napari/_tests/test_function_widgets.py E                                                       [ 27%]
napari/_tests/test_key_bindings.py EEE                                                         [ 29%]
napari/_tests/test_mouse_bindings.py EEE                                                       [ 31%]
napari/_tests/test_notebook_display.py E                                                       [ 31%]
napari/_tests/test_numpy_like.py EEEEEEEE                                                      [ 36%]
napari/_tests/test_view_layers.py ..............EEEEEEEEEEEEEEE.                               [ 54%]
napari/_tests/test_viewer.py EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEsE   [ 92%]
napari/_tests/test_with_screenshot.py sssssssssssss                                            [100%]

=============================================== ERRORS ===============================================
```

Reply:
(looks like a lot of stuff in `test_view_layers` is fine without qtpy)

Reply:
yeah, ok. makes sense, thanks!